docs/source/essentials/custom-scalars.mdx

```diff
@@ -14,13 +14,13 @@ To interact with custom scalars in your Apollo Kotlin app, you need to define a
 ```kotlin:title=build.gradle[.kts]
 apollo {
   mapScalar("GeoPoint", "com.example.GeoPoint")
-  
+
   // Shortcuts exist for standard types - equivalent to mapScalar("Long", "kotlin.Long")
   mapScalarToKotlinLong("Long")
 }
 ```
 
-Note: if needed, you can also do this with built-in scalars (like `ID`) to override the type used by default.
+> If needed, you can also do this with _default_ scalars (like `ID`) to override the type used by default.
```

Comment:
Nit: I like "built-in" better because this is the terminology from the spec: https://spec.graphql.org/draft/#sec-Scalars